### PR TITLE
Add comment to client_info

### PR DESCRIPTION
**Pull request by bobzilladev**
Modeling off of [ngrok-go](https://github.com/ngrok/ngrok-go/blob/73d54eba1716e8889b7034f8ecd8071d69f8691a/session.go#L192).
Semver check fails, but this is a currently undocumented method, hopefully ok?

May want it to be `Option<Vec<String>>` to remove the `impl Into` so don't have to qualify a `None` like `None::<Vec<String>>`. Loses the flexibility, but makes the common case simpler.

### CODE REVIEW

**Comment by bobzilladev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #92** <a href="https://app.graphite.dev/github/pr/ngrok/ngrok-rs/92" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ngrok/ngrok-rs/92?utm_source=stack-comment).

File: ngrok/src/session.rs [hunk 1]
```diff
@@ -498,9 +498,13 @@ impl SessionBuilder {
         mut self,
         client_type: impl Into<String>,
         version: impl Into<String>,
+        comments: Option<Vec<impl Into<String>>>,
```
Comment (jrobsonchase) [inline ngrok/src/session.rs:501]: I think we can leave it as `Option<impl Into<String>>`. The varargs in the Go implementation were just a way to fake optional arguments without having to resort to nullable pointers or checking for an empty string.

**Review by jrobsonchase — CHANGES_REQUESTED** [review]

File: ngrok/src/session.rs (hunk 1 above)
Comment (bobzilladev) [reply ngrok/src/session.rs]: Pushed a commit with this change, and also speculatively renamed to `client_info` and removed the doc hiding.

**Review by jrobsonchase — APPROVED** [review]
Drop that `sanitize_ua_string` call from the comment and I think this is good to go 👍

File: ngrok/src/session.rs [hunk 2]
```diff
@@ -591,11 +587,14 @@ impl SessionBuilder {
...
+            .map(|(name, version, comments)| {
                 format!(
-                    "{}/{}",
+                    "{}/{}{}",
                     sanitize_ua_string(name),
-                    sanitize_ua_string(version)
+                    sanitize_ua_string(version),
+                    comments
+                        .as_ref()
+                        .map_or(String::new(), |f| format!(" ({})", sanitize_ua_string(f)))
```
[the remark this answers is not in the document]
Comment (bobzilladev) [reply ngrok/src/session.rs:597]: Nevermind, error was from elsewhere, will do!

**Pull request merged**